app/views/dashboard.pug

```diff
@@ -38,7 +39,9 @@ html(lang="en")
                         div(style='position:absolute;width:100%;height:20%;left:0%;top:80%;')
                             button.btn.btn-default(type='submit' style='width:100%;height:100%;left:0%;top:0%;') Logout
 
-                div(id='mainDiv' style='position:absolute;width:90%;height100%;left:10%;top:0%;border-style:solid;')
+                div(id='mainDiv' style='position:absolute;width:90%;height100%;left:10%;top:0%;border-style:solid; z-index:1')

```

Comment:
Ok, what glitches occurred when we tried to overlay the mainDiv with the pop up?